tests/src/Functional/ParagraphsTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_paragraphs\Functional;
+
+use Drupal\Core\Url;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests paragraphs forms.
+ */
+class ParagraphsTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'node',
+    'field_ui',
+    'oe_paragraphs',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
```

Comment:
we will be only accessing the admin theme

Reply:
I need this or I get this error: 
1) Drupal\Tests\oe_paragraphs\Functional\ParagraphsTest::testFactsAndFiguresParagraph
Exception: Drupal\Tests\BrowserTestBase::$defaultTheme is required. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.